### ISSUE TITLE
ecspold v2: Better dataset accessor

### ISSUE THREAD

**Issue by cmutel** [sami-m-g/pyecospold#34]
To get a dataset, we currently have to do something like:

```python
    try:
        ds = ecospold.childActivityDataset
        assert ds
    except (AttributeError, AssertionError):
        ds = ecospold.activityDataset
```

You would be doing this every time we use this library, and it sucks. So I think it is completely fine to have this code internally (maybe rewritten), and only expose the `activityDataset` property, which can return either a `childActivityDataset` and an `activityDataset`, and can also be set with both of these classes.

**Comment by sami-m-g** [comment]
@cmutel is this similar to #28 ?

**Comment by sami-m-g** [comment]
@cmutel resolved by 8ba621a.
